docs/linting/Troubleshooting.mdx

```diff
@@ -30,21 +30,67 @@ If you don't find an existing extension rule, or the extension rule doesn't work
 
 ## I get errors telling me "ESLint was configured to run ... However, that TSConfig does not / none of those TSConfigs include this file"
 
+These errors are caused by an ESLint config requesting type information be generated for a file that isn't included in the TypeScript configuration.
+
 ### Fixing the Error
 
+<!-- prettier-ignore-start -->
+
 - If you **do not** want to lint the file:
   - Use [one of the options ESLint offers](https://eslint.org/docs/latest/user-guide/configuring/ignoring-code) to ignore files, namely a `.eslintignore` file, or `ignorePatterns` config.
 - If you **do** want to lint the file:
   - If you **do not** want to lint the file with [type-aware linting](./Typed_Linting.mdx):
-    - Use [ESLint's `overrides` configuration](https://eslint.org/docs/latest/user-guide/configuring/configuration-files#configuration-based-on-glob-patterns) to configure the file to not be parsed with type information.
-      - A popular setup is to omit the above additions from top-level configuration and only apply them to TypeScript files via an override.
-      - Alternatively, you can add `parserOptions: { project: null }` to an override for the files you wish to exclude. Note that `{ project: undefined }` will not work.
+    - Use [ESLint's `overrides` configuration](https://eslint.org/docs/latest/user-guide/configuring/configuration-files#configuration-based-on-glob-patterns) to configure the file to not be parsed with type information:
+      <details>
+        <summary>
+          A popular setup is to remove all rules requiring type information from the top-level configuration
+          and only apply them to TypeScript files via an override.
+        </summary>
+
+        ```js title=".eslintrc.cjs"
+        module.exports = {
+          // ... the rest of your config ...
+          overrides: [
+            {
+              extends: [
+                'plugin:@typescript-eslint/recommended-requiring-type-checking',
+              ],
+              files: ['./**/*.{ts,tsx}'],
+            },
+          ],
+        };
+        ```
+
+      </details>
+      <details>
```

Comment:
Maybe out this first so that the simpler config option is front and Center?

Reply:
Hmm interesting! Maybe once v6 lands we can? I personally lean towards side of putting additive changes in `overrides`. So maybe soon 😄